connector/src/main/java/org/jboss/as/connector/subsystems/common/pool/PoolOperations.java

```diff
@@ -112,6 +112,17 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
         }
     }
 
+    private String concatenateExceptionCauses(Exception e) {
+        StringBuilder sb = new StringBuilder();
+        sb.append(e.getMessage());
+        Throwable t = e.getCause();
+        while (t != null) {
+            sb.append(" ").append(t.getMessage());
```

Comment:
@tadamski WDYT about using toString() here, and perhaps at L117? That adds the exception class name.

I've used getMessage() a lot over the years but then noticed when I looked at logs I didn't understand the problem because the message writer kind of assumed there'd be more info to provide context.

Also, perhaps instead of sb.append(" ") go with sb.append(" Caused by: ")?

Reply:
originally `e.getLocalizedMessage()` was used, is it ok to change that to simpler `e.getMessage()` or `e.toString()`? IOW are we ok we are losing i18n here?

Reply:
@bstansberry Seems good to me, I have fixed the code to use toString method
@jbliznak Looking at the JDK code, it seems that to String is used getLocalizedMessage under the hood anyway so we shouldn't lose the localization